addons/web_editor/static/src/js/backend/field_html.js

```diff
@@ -53,7 +53,33 @@ var FieldHtml = basic_fields.DebouncedField.extend(TranslatableFieldMixin, Dynam
         wysiwyg_change: '_onChange',
         wysiwyg_attachment: '_onAttachmentChange',
     },
+    /**
+     * @override
+     * @param {ModelFieldSelectorPopover} modelSelector
+     * @returns {Promise<ModelFieldSelectorPopover>}
+     */
+    insertAndPositionModelSelector: async function (modelSelector) {
```

Comment:
The docstring of this function really should mention that it positions the model selector with respect to the range. That is a very important behavioral change compared to the "standard" behavior in the basic field and that is an actual feature.